.github/workflows/preDeploy.yml

```diff
@@ -44,11 +44,29 @@ jobs:
           GITHUB_TOKEN: ${{ github.token }}
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
 
+  # Get details about the merged pull request
+  getMergedPullRequest:
+    runs-on: ubuntu-latest
+
+    outputs:
+      title: ${{ steps.getMergedPullRequest.outputs.title }}
+      body: ${{ steps.getMergedPullRequest.outputs.body }}
+      number: ${{ steps.getMergedPullRequest.outputs.number }}
+      labels: ${{ steps.getMergedPullRequest.outputs.labels }}
+      assignees: ${{ steps.getMergedPullRequest.outputs.assignees }}
+      author: ${{ steps.getMergedPullRequest.outputs.author }}
```

Comment:
Thanks for handling the revert. I have tried to test this but at the moment, I have not quickly created required files. [Here](https://github.com/Expensify/App/pull/8193) is a draft PR with a follow up.

For some reason the failed actions seem like it has not assigned `labels` to the outputs but probably any other. My guess is that the steps here must be wrapped in fromJSON to properly pass its value. 

I will try to test this by creating a new workflow on merge and have a dummy PR merge it and see if the values are assigned correctly there just to make sure we do not have to revert anything there.

Reply:
@mountiny I also recommend that you try testing in https://github.com/Andrew-Test-Org/Public-Test-Repo, which is what I always do to workshop workflows I'm unsure about. I can give you admin access to merge your own PRs. Another option would be to try and get [act](https://github.com/nektos/act) working for a local test.

Reply:
@roryabraham Ah, that is great! Didn't know about this testing org, I will try to utilize that. Act looks nice too, do you use it or do you prefer the test org for things you are unsure about?